apps/test/util/testUtils.js

```diff
@@ -261,14 +273,26 @@ export function throwOnConsoleErrorsEverywhere() {
   let firstError = null;
   throwingOnErrors = true;
   beforeEach(function () {
+    // Stash test title so that we can include it in any errors
+    let testTitle;
+    if (this.currentTest) {
+      testTitle = this.currentTest.title;
+    }
+
     sinon.stub(console, 'error').callsFake(msg => {
       const prefix = throwingOnErrors ? '' : '[ignoring]';
       console.error.wrappedMethod(prefix, msg);
+      if (throwingOnErrors) {
+        logStack();
```

Comment:
Is there a reason we're printing the stack right away, instead of storing the it and printing it with the stored error after the test?  In fact, why not attach the stack to our `firstError` object?

Just suggestions, this is clearly an improvement without it.